### ISSUE TITLE
includePolyfill option

### ISSUE THREAD

**Issue by ilkkao** [babel/ember-cli-babel#61]
I just upgraded ember-cli from 1.13.8 to master and started to get error:

```
Build failed.
File: ember-cli-app-version/components/app-version.js
[BABEL] ember-cli-app-version/components/app-version.js: Unknown option: direct.includePolyfill
```

It goes away if I remove `includePolyfill: true` from ember-cli-build.js, but I need the polyfill. It looks like the addon removes `options.includePolyfill` so not sure at all why this is an issue.

**Comment by ilkkao** [comment]
During `ember init` I accepted all bower and npm module version changes.

**Comment by mwpastore** [comment]
I'm having the same issue after upgrading to 1.13.10. Is there a new way to enable the polyfill or is it just enabled by default now?

**Comment by stefanpenner** [comment]
> Is there a new way to enable the polyfill or is it just enabled by default now?

sounds like a bug

**Comment by ivanvotti** [comment]
the same here: `[BABEL] ember-ajax/errors.js: Unknown option: direct.includePolyfill`

```
$ ember s
version: 1.13.11
[Deprecated] new this.Funnel(..) is deprecated, please use broccoli-funnel directly instead  [addon: liquid-fire]
Livereload server on http://localhost:49154
Serving on http://localhost:4200/
File: ember-ajax/errors.js
[BABEL] ember-ajax/errors.js: Unknown option: direct.includePolyfill
```

When I commented out `includePolyfill`, there's no error.

**Comment by topaxi** [comment]
This happens to me in ember-cli 1.13.11 now too, worked on 1.13.8 though.

**Comment by machty** [comment]
I believe the fix is as simple as changing `includePolyfill: true` to `browserPolyfill: true`

**Comment by Dhaulagiri** [comment]
@machty that makes the error go away but I don't see the polyfill actually being included in my vendor.js

**Comment by Dhaulagiri** [comment]
Played around with this some more and opened #63 with a fix that worked locally

**Comment by ivanvotti** [comment]
I tried [Dhaulagiri's fix](https://github.com/babel/ember-cli-babel/pull/63) and it fixes the problem for me. The polyfill is included and the error is gone.

**Comment by mwpastore** [comment]
Confirmed. After applying the patch to ember-cli-babel:

With `browserPolyfill: false`:

```javascript
$ grep -i babel vendor.js 
  // https://github.com/babel/babel/issues/906 is resolved
    _emberMetalCore.default.deprecate('Ember.String.fmt is deprecated, use ES6 template strings instead.', false, { id: 'ember-string-utils.fmt', until: '3.0.0', url: 'https://babeljs.io/docs/learn-es6/#template-strings' });
      @deprecated Use ES6 template strings instead: https://babeljs.io/docs/learn-es6/#template-strings');
```

With `browserPolyfill: true`:

```javascript
$ grep -i babel vendor.js 
if (global._babelPolyfill) {
  throw new Error("only one instance of babel/polyfill is allowed");
global._babelPolyfill = true;
  // https://github.com/babel/babel/issues/906 is resolved
    _emberMetalCore.default.deprecate('Ember.String.fmt is deprecated, use ES6 template strings instead.', false, { id: 'ember-string-utils.fmt', until: '3.0.0', url: 'https://babeljs.io/docs/learn-es6/#template-strings' });
      @deprecated Use ES6 template strings instead: https://babeljs.io/docs/learn-es6/#template-strings');
```

**Comment by pixelhandler** [comment]
@Dhaulagiri @ivanvotti thanks of linking up the fix in ember-cli-babel, fixes my build issue after upgrading ember-cli from 1.13.8 to 1.13.12

**Comment by Dhaulagiri** [comment]
This is fixed now with ember-cli 1.13.13
